gbpservice/nfp/bin/nfp_configurator.ini

```diff
@@ -17,6 +17,9 @@ modules_dir=gbpservice.nfp.configurator.modules
 reportstate_interval=10
 periodic_interval=9
 
+log_forward_ip_address=
+log_forward_port=514
```

Comment:
The one who changes this must be aware that the new port is listened to by
the Visibility VM. We just forwards logs to this port.

Regards,
Deepak

On Mon, May 2, 2016 at 2:42 PM, njagadish <notifications@github.com> wrote:

> In gbpservice/nfp/bin/nfp_configurator.ini
> <https://github.com/oneconvergence/group-based-policy/pull/271#discussion_r61718096>
> :
>
> > @@ -17,6 +17,9 @@ modules_dir=gbpservice.nfp.configurator.modules
> >  reportstate_interval=10
> >  periodic_interval=9
> >
> > +log_forward_ip_address=
> > +log_forward_port=514
>
> If we change this port, will things work on the configured (other than
> 514) port?
>
> —
> You are receiving this because you authored the thread.
> Reply to this email directly or view it on GitHub
> <https://github.com/oneconvergence/group-based-policy/pull/271/files/3173d44dcc0b5e61ffcb486e9c295160ac1fd6a0#r61718096>
>